### DISCUSSION TITLE
Some sopel's old plugins downgrade Sopel via pip command.

### DISCUSSION THREAD

**Discussion by antdude** [sopel-irc/sopel#2184] [Q&A] [answered]
https://pypi.org/search/?q=%22sopel%22 shows old plugin like https://pypi.org/project/sopel_modules.habitica/ that hasn't been updated since 5/18/2016. I noticed my Sopel was downgraded to v6.6.9 from v7.1.4 after installing a bunch of Sopel plugins like Habitca via pip command. :(

There needs to a way to prevent downgrading Sopel with old plugins. Just deny these old plugins that want older versions. 

Thank you for reading and hopefully finding a way to do this.

**Comment by dgw** [comment, marked as answer]
One of the [answers here](https://stackoverflow.com/q/43121005/5991).

This is a pip question, not a Sopel question.

**Reply by antdude** [reply]
Interesting and thanks. I wished it was like apt-get.
